psyneulink/library/compositions/pytorchcomponents.py

```diff
@@ -0,0 +1,276 @@
+from psyneulink.core.components.functions.transferfunctions import Linear, Logistic, ReLU
+from psyneulink.library.compositions.pytorchllvmhelper import *
+from psyneulink.core.globals.log import LogCondition
+from psyneulink.core import llvm as pnlvm
+
+try:
+    import torch
+    from torch import nn
+    torch_available = True
+except ImportError:
+    torch_available = False
+
+
+def pytorch_function_creator(function, device, context=None):
+    """
+    Converts a PsyNeuLink function into an equivalent PyTorch lambda function.
+    NOTE: This is needed due to PyTorch limitations (see: https://github.com/PrincetonUniversity/PsyNeuLink/pull/1657#discussion_r437489990)
+    """
+    def get_fct_param_value(param_name):
+        val = function._get_current_function_param(
+            param_name, context=context)
+        if val is None:
+            val = getattr(function.defaults, param_name)
```

Comment:
It appears so - `_get_current_function_param` doesn't seem to return the default value (not sure if that is intended).

Reply:
ok, thanks for checking

Reply:
I checked parameter `_get` and it needs `parameter.fallback_default` to get that behaviour. is that not the case for these parameters? if not, should it be?

Reply:
It doesn't seem to be the case - `return getattr(self.parameters, param_name)._get(context)` seems to be returning `None` instead of the default value of the parameter (seems to be due to contexts). Seems like a bug.